Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -0,0 +1,75 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
+import Result
+import UIKit
+
+final class SettingsAccountViewController: UIViewController {
+
+  @IBOutlet private weak var tableView: UITableView!
+
+  private let dataSource = SettingsAccountDataSource()
+  fileprivate let viewModel: SettingsAccountViewModelType = SettingsAccountViewModel()
+
+  internal static func instantiate() -> SettingsAccountViewController {
+    return Storyboard.SettingsAccount.instantiate(SettingsAccountViewController.self)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.tableView.dataSource = dataSource
+    self.tableView.delegate = self
+
+    self.tableView.register(nib: .SettingsTableViewCell)
+    self.tableView.registerHeaderFooter(nib: .SettingsHeaderView)
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  override func bindViewModel() {
+    self.viewModel.outputs.reloadData
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.dataSource.configureRows()
+        self?.tableView.reloadData()
+    }
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseControllerStyle()
+      |> UIViewController.lens.view.backgroundColor .~ .ksr_grey_200
```

Comment:
I think we use this styling in enough VCs to warrant it getting its own style property in `SettingsStyles.swift`. Maybe something like `baseSettingsControllerStyle`?